src/xdp-utils.c

```diff
@@ -815,7 +822,7 @@ xdp_filter_options (GVariant *options,
 
           continue;
         }
-         
+
```

Comment:
Unrelated whitespace cleanup

Reply:
I'm okay with this unrelated cleanup :slightly_smiling_face: